x/participation/keeper/msg_withdraw_allocations_test.go

```diff
@@ -165,7 +167,12 @@ func Test_msgServer_WithdrawAllocations(t *testing.T) {
 			// check usedAllocationEntry is correctly decreased
 			postUsedAllocations, found := tk.ParticipationKeeper.GetUsedAllocations(tmpSdkCtx, tt.msg.Participant)
 			require.True(t, found)
-			require.Equal(t, preUsedAllocations.NumAllocations-preAuctionUsedAllocations.NumAllocations, postUsedAllocations.NumAllocations)
+			calculated := preUsedAllocations.NumAllocations.Sub(preAuctionUsedAllocations.NumAllocations)
+			if calculated.IsZero() {
+				require.True(t, postUsedAllocations.NumAllocations.IsZero())
+				return
+			}
+			require.Equal(t, calculated, postUsedAllocations.NumAllocations)
```

Comment:
You can do this with a single assertion :

```suggestion
			require.Equal(t, calculated.Int64(), postUsedAllocations.NumAllocations.Int64())
```

Or if you prefer to not call `Int64()` :

```suggestion
			require.True(t, postUsedAllocations.NumAllocations.Equal(calculated), "numAlloc %s not equal to calculated %s", postUsedAllocations.NumAllocations, calculated)
```